addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -502,6 +515,12 @@ export class LinkPlugin extends Plugin {
                             (FONT_SIZE_CLASSES.some((cls) => el.classList.contains(cls)) ||
                                 el.style?.fontSize)
                     );
+                    if (classes) {
+                        link.className = classes;
+                    }
```

Comment:
This seems unrelated to the change? The `rel` sure, but the classes? Is this a fix for an unrelated issue that you noticed while working on this PR ?